### DISCUSSION TITLE
osu!taiko Swap Mod as FreeMod Crashes in Multiplayer

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16729] [Q&A] [answered]
Initially shown in https://youtu.be/DDVeIYY3pYw?t=695 but replicated in the current version of 2022.128.0 for logs.
This doesn't occur when adding Swap globally, only when available as a FreeMod. I'd wager that this is due to the incorrect ruleset being applied and thus can be replicated with any mod not available in std, but I don't want to open a bajillion lobbies to find out

https://user-images.githubusercontent.com/74262407/151927968-4e69c5c3-1fe8-4b70-869d-28d44189119a.mp4

Applicable portions of [runtime.log.txt](https://github.com/ppy/osu/files/7976562/runtime.log.txt)
.

**Comment by peppy** [comment, marked as answer]
Tracking via #16730.
